eloquentjs/5_3_everything.js

```diff
@@ -0,0 +1,88 @@
+function everythingSome(inputArray, testFunction) {
+    const antiTest = j => !testFunction(j);
+    return inputArray.some(testFunction) && !inputArray.some(antiTest);
+}
+
+function everythingLoop(inputArray, testFunction) {
+    for (let element of inputArray) {
+        if (!testFunction(element)) {
+            return false
+        }
+    }
+    return true;
+}
+
+const greaterThanN = n => (x => x > n);
+const greaterThanZero = greaterThanN(0);
+const greaterThanTwo = greaterThanN(2);
+const greaterThanTen = greaterThanN(10);
+const greaterThanNegativeFifty = greaterThanN(-50);
+const greaterThanFifty = greaterThanN(50);
+
+const testCases = [
+    [0, 1, 2, 3],
+    [1, 2, 3],
+    [-2, -1, 0, 1, 2],
+    [-2, -1, 0],
+    [-2, -1],
+    [0],
+    [10, 11, 12],
+    [-12, -11, -10],
+    [65, -766, 432, -1, 8],
+];
+
+const testFunctionDictionary = {
+    greaterThanZero: greaterThanZero,
+    greaterThanTwo: greaterThanTwo,
+    greaterThanTen: greaterThanTen,
+    greaterThanNegativeFifty: greaterThanNegativeFifty,
+    greaterThanFifty: greaterThanFifty,
+}
+
+console.log("Testing Everything Some...\n");
+
+let passesAllTests = true;
+
+for (let testArray of testCases) {
```

Comment:
I really like the approach you're taking of using the built in function to test a ton of cases, but think we can clean up the implementation a bit.
1) You could factor the test runner logic for both of your implementations into a higher order function
2) A lot of signal will be lost in the sheer volume of printing, I'd recommend just printing something like a checkmark when a test passes so failures stick out. A message like `Failure: everything([-2, -1], greaterThanTwo) === true` would be sufficient.